spec.bs

```diff
@@ -325,13 +325,32 @@ spec: html; urlPrefix: https://www.rfc-editor.org/rfc/
     1. Return |result|.
   </div>
 
+  <h2 id="get-the-number-of-distinct-versions-in-epochs-header">Get the number of distinct versions in epochs</h2>
+  <div algorithm>
+    To <dfn>get the number of distinct versions in epochs</dfn>, given a [=topics caller context=] |callerContext|, perform the following steps. They return an integer.
+    1. Let |epochs| be the result of running the [=calculate the epochs for caller=] algorithm given |callerContext| as input.
+    1. Let |distinctVersions| be an empty set.
+    1. For each |epoch| in |epochs|:
+        1. If |epoch|'s [=epoch/taxonomy version=] is empty (implying that the topics calculation for that epoch didn't occur), then continue.
+        1. Insert tuple (|epoch|'s [=epoch/taxonomy version=], |epoch|'s [=epoch/model version=]) to distinctVersions.
+    1. Return |distinctVersions|'s [=list/size=].
+  </div>
+
   <h2 id="topics-for-caller-header">Topics for caller</h2>
   <div algorithm>
-    To <dfn>calculate the topics for caller</dfn>, given a [=topics caller context=] |callerContext|, perform the following steps. They return a list of {{BrowsingTopic}}s.
+    To <dfn>calculate the topics for caller</dfn>, given a [=topics caller context=] |callerContext|, perform the following steps. They return a list of {{BrowsingTopic}}s
```

Comment:
Nit: the period was good, keep the period :)

Reply:
Done.